adm_program/system/classes/Email.php

```diff
@@ -565,6 +577,32 @@ public function setTemplateText($text, $senderName, $senderEmail, $senderUuid, $
         $this->emHtmlText = $emailHtmlText;
     }
 
+
+    /**
+     * Add the user specific template text to the email message and replace the plaeholders of the template.
+     * @param string $text        Email text that should be send
+     * @param string $firstname  Receiver firstname
+     * @param string $surname Receiver surname
+     * @param string $email  Receiver email address
+     * @param string $name  Receiver firstname and surname
+     */
+    public function setUserSpecificTemplateText($text, $firstName, $surname, $email, $name)
+    {
+        // replace all line feeds within the mailtext into simple breaks because only those are valid within mails
+        $text = str_replace("\r\n", "\n", $text);
+
+        // replace parameters in email template
+        $replaces = array(
+            '#receiver_first_name#' => $firstName,
```

Comment:
We should only use one parameter for firstname and one parameter for surname. 
Maybe `#receiver_firstname#` and `#receiver_lastname#` are good ones.